src/hotspot/share/runtime/os.hpp

```diff
@@ -770,6 +770,9 @@ class os: AllStatic {
   // Get host name in buffer provided
   static bool get_host_name(char* buf, size_t buflen);
 
+  // Returns the process working set size (rss); 0 if unsupported.
+  static size_t get_rss();
```

Comment:
Nit: as it is an acronym `get_RSS` would be better IMO - just for this accessor; no need to rename everything to RSS.